Missionframework/functions/curator/fn_initCuratorHandlers.sqf

```diff
@@ -0,0 +1,87 @@
+/*
+    File: fn_initCuratorHandlers.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2020-08-07
+    Last Update: 2020-08-07
+    License: MIT License - http://www.opensource.org/licenses/MIT
+
+    Description:
+        Initialize curator handlers.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+#define ZEUSVAR(var) (format ["KPLIB_zeus_%1", var])
+
+if (isServer) then {
+
+    // delete old Liberation mission placed Zeus module
+    deleteVehicle zm1;
+
+    // add curator assign EventHandler
+    [true, "KPLIB_createZeus", {
+        params [
+            ["_player", objNull, [objNull]],
+            ["_limited", false, [true]]
+        ];
+
+        if (isNull _player) exitWith {};
+
+        // remove currently assigned curator
+        private _oldZeus = getAssignedCuratorLogic _player;
+        unassignCurator _oldZeus;
+        deleteVehicle _oldZeus;
+
+        private _uid = getPlayerUID _player;
+        private _group = createGroup sideLogic;
+        _group deleteGroupWhenEmpty true;
+        private _zeus = _group createUnit ["ModuleCurator_F", [-7580, -7580, 0], [], 0, "NONE"];
+        missionNamespace setVariable [ZEUSVAR(_uid), _zeus];
+
+        if (_limited) then {
+            _zeus setVariable ["owner", _uid, true];
+            _zeus setVariable ["Addons", 0, true];
+            _zeus setVariable ["BIS_fnc_initModules_disableAutoActivation", false];
```

Comment:
`owner` and `disableAutoActivation` can be moved outside of the control structure, as it's doubled in then and else case.